src/Api/Controllers/AuthRequestsController.cs

```diff
@@ -37,6 +37,15 @@ public class AuthRequestsController : Controller
             _pushNotificationService = pushNotificationService;
         }
 
+        [HttpGet("")]
+        public async Task<ListResponseModel<AuthRequestResponseModel>> Get()
+        {
+            var userId = _userService.GetProperUserId(User).Value;
```

Comment:
Should this be `?.Value` in case it returns `null`?

Reply:
I don't think so. That isn't the pattern used when checking `GetProperUserId()` elsewhere (something that is done often), and this is an endpoint that can't be hit unless we're working with an authenticated session, so we would want an error to throw if that value was null here.

I could see wanting to improve the error handling, but anyone that makes it this far without an authenticated session is likely a bad actor, and I don't think we're obligated to give those folks useful error messages.